### PR TITLE
Deprecated all operations.

### PR DESCRIPTION
**Pull request by lfalck**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

### CODE REVIEW

File: certified-connectors/Integration Toolbox/apiDefinition.swagger.json [hunk 1]
```diff
@@ -246,9 +250,10 @@
     },
     "/pgpdecrypt": {
       "post": {
-        "operationId": "PGPDecrypt",
+        "operationId": "PGPDecrypt (deprecated)",
```
Comment (Amjed-Ayoub) [inline certified-connectors/Integration Toolbox/apiDefinition.swagger.json:253]: @lfalck the operationId is changes here. I think you meant to change the summary instead of this?
It is not allowed to change the operationIds. Please revert this change.

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

**Comment by lfalck** [comment]
Sorry about that, I fixed the issue now!
